airqo-platform/AirQo-frontend
# Issues #74 device status summary

What does this PR do?
This PR fixes issue https://github.com/airqo-platform/AirQo-frontend/issues/74
What JIRA task is related to this PR?
NOT Jira task
How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate`. Then run `npm install` to get all the necessary dependencies. Afterwards, run `npm run local`.
Which endpoints should are ready for testing?
None
